GameOfRunes/GameOfRunes/GameEngine/Entities/EnemyEntity.swift

```diff
@@ -38,9 +40,15 @@ class EnemyEntity: Entity {
         )
  
         let spriteComponent = SpriteComponent(node: enemyNode, layerType: .unitLayer)
+
+        // Monster Speed and Acceleration are Proportionate to Screen Size (Treat speed as ratio)
+        let timetaken = GameConfig.Unit.standardUnitTraversalTime
+        let enemyMaxSpeed = ((enemyType.speed / 100.0) * Float(renderNodeSize.height)) / timetaken
+        let enemyMaxAcceleration = GameConfig.Unit.accelerationScalingFactor * enemyMaxSpeed
```

Comment:
does this affect the freeze powerup as well?

Reply:
Okay resolved!